src/libs/protocols/protos/agent.proto

```diff
@@ -444,6 +448,23 @@ message FSGroup {
 	types.FSGroupChangePolicy group_change_policy = 3;
 }
 
+// SharedMount declares a set of shared bind mount points that support
+// cross-container sharing of mount objects. Including the mount point's name,
+// path and type is necessary.
+message SharedMount {
```

Comment:
Let's just pass `config::SharedMount` to the agent in `CreateContainerRequest` for the destination container.